src/index.js

```diff
@@ -80,6 +82,53 @@ async function main ( args ) {
 	    });
 	}));
 
+	commander
+	.command("install-config <path/to/app-config.yml>")
```

Comment:
See other comment for reasoning
```suggestion
	.command("setup-apps <path/to/app-config.yml>")
```